Sources/Core/Microsoft.StreamProcessing/Operators/Afa/UngroupedDAfaTemplate.cs

```diff
@@ -14,7 +14,7 @@ namespace Microsoft.StreamProcessing
     /// <summary>
     /// Class to produce the template output
     /// </summary>
-    [global::System.CodeDom.Compiler.GeneratedCodeAttribute("Microsoft.VisualStudio.TextTemplating", "16.0.0.0")]
+    [global::System.CodeDom.Compiler.GeneratedCodeAttribute("Microsoft.VisualStudio.TextTemplating", "15.0.0.0")]
```

Comment:
Looks like someone hasn't upgraded to Visual Studio 2019 yet? :-)

Reply:
I'll start it now, unless you think it's needed for this change

---
In reply to: [286226689](https://github.com/microsoft/Trill/pull/94#discussion_r286226689) [](ancestors = 286226689)

Reply:
Nah, just for the future.